tests/unit/console_test.ts

```diff
@@ -2118,30 +2166,6 @@ Deno.test(function inspectProxy() {
     )),
     `{ key: "value" }`,
   );
-  assertEquals(
```

Comment:
Removed this test, because it was asserting that we _did_ invoke proxy traps, even if we shouldn't.